### ISSUE TITLE
Add USB LiPo Charge Circuit?

### ISSUE THREAD

**Issue by NeilKhera** [PyCubed-Mini/avionics-mainboard#26]
Could be useful. The full-sized PyCubed board has this

**Comment by jbwillis** [comment]
Yeah this could be nice. There are reasons not to though. Mostly so we can monitor the satellite without changing its power state.
